api-inference-community/manage.py

```diff
@@ -63,6 +65,30 @@ def show(args):
                         print(" " * 4, key.value)
 
 
+def resolve(model_id: str) -> [str, str]:
+    info = HfApi().model_info(model_id)
+    task = info.pipeline_tag
+    framework = None
+    frameworks = {
```

Comment:
Nit: please sort `frameworks` alphabetically :)

Reply:
Ahaah is that really that important ?  :D